RELEASE

```diff
@@ -4,6 +4,30 @@
  / ___ |/ /_/ / /_/ /___/ / /__/ /_/ / //  __/
 /_/  |_/ .___/ .___//____/\___/\__,_/_/ \___/ 
       /_/   /_/                               
+AppScale version 2.4.0, released September, 2015
+Highlights of features/bugs in this release:
+- Upgraded to Ruby 1.9
+- Upgraded Java support to SDK 1.8.1
+- Fixed Datastore bug: fetching fewer references than the existing
+- Fixed Datastore bug for values containing a reserved delimiter
+- Better index validation when fetching entities
```

Comment:
Since we didn't need to validate indexes before, this should probably be "Fixed datastore transactions" or something.